src/shared/components/FullTextSearch/useFullTextSearch.ts

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react';
+import { useNexusContext } from '@bbp/react-nexus';
+import { useQuery } from 'react-query';
+import { getOrgAndProjectFromProjectId } from 'shared/utils';
+import { groupBy } from 'lodash';
+
+export function useFullTextSearch() {
+  const [search, setSearch] = useState('');
+  const nexus = useNexusContext();
+
+  const onSearch = (value: string) => setSearch(value);
+  const resetSearch = () => setSearch('');
+
+  const { isLoading, data } = useQuery({
+    enabled: !!search,
+    queryKey: ['cmdk-search', { search }],
+    queryFn: () =>
+      nexus.Resource.list(undefined, undefined, {
+        q: search,
+        deprecated: false,
+      }),
+    select: data => data._results,
+    staleTime: 2,
```

Comment:
The `staleTime` in milliseconds right? 2 milliseconds sounds too low to me. Did you mean 2000 ms instead?